src/api/app/controllers/attribute_controller.rb

```diff
@@ -34,59 +35,60 @@ def index
     render xml: xml
   end
 
-  # /attribute/:namespace/_meta
-  def namespace_definition
+  def ensure_namespace
     if params[:namespace].nil?
       raise MissingParameterError, "parameter 'namespace' is missing"
     end
-    namespace = params[:namespace]
+    params[:namespace]
+  end
 
-    if request.get?
-      @an = AttribNamespace.where(name: namespace).select(:id, :name).first
-      unless @an
-        render_error message: "Unknown attribute namespace '#{namespace}'",
-          status: 404, errorcode: 'unknown_attribute_namespace'
-      end
-      return
+  def show_namespace_definition
+    @an = AttribNamespace.where(name: ensure_namespace).select(:id, :name).first
+    unless @an
+      render_error message: "Unknown attribute namespace '#{namespace}'",
+        status: 404, errorcode: 'unknown_attribute_namespace'
     end
+  end
 
+  def require_admin
```

Comment:
This is already defined [here](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/application_controller.rb#L33)

Reply:
@coolo Ah. I didn't see that there was an admin check in some controller already.

In that leave it as is. Let's not change the API behaviour.

Reply:
I reconsidered and I don't think anyone relies on the different error message for requiring admin.